### PR TITLE
Add networkType to Self-Managed aws

### PR DESCRIPTION
**Pull request by krishvoor**
This PR allows users to set `networking.networkType`
as OVNKubernetes/OpenShiftSDN, defaults to OVNKubernetes

### CODE REVIEW

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.15-nightly-x86-control-plane-24nodes-openshiftsdn

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml [hunk 1]
```diff
@@ -50,6 +50,18 @@ tests:
...
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      NETWORK_TYPE: OpenShiftSDN
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:64]: https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/aws/sdn/pre/ipi-aws-sdn-pre-chain.yaml 

Should we just call this ?

Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml:64]: Yeah right, we can do the same stuff via this https://github.com/openshift/release/blob/master/ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh#L6

**Comment by krishvoor** [comment]
Closing this PR - as we have a ref in place.

**Pull request closed without merging**